tests/expose_test.go

```diff
@@ -81,35 +97,104 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		Context("Expose ClusterIP service", func() {
 			const servicePort = "27017"
 			const serviceName = "cluster-ip-vmi"
-			It("[test_id:1531][label:masquerade_binding_connectivity]Should expose a Cluster IP service on a VMI and connect to it", func() {
+
+			var vmiExposeArgs []string
+
+			var jobCleanupFunc func() error
+			var serviceCleanupFunc func() error
+
+			BeforeEach(func() {
+				vmiExposeArgs = []string{
+					expose.COMMAND_EXPOSE,
+					"virtualmachineinstance", "--namespace", tcpVM.GetNamespace(), tcpVM.GetName(),
+					"--port", servicePort, "--name", serviceName,
+					"--target-port", strconv.Itoa(testPort),
+				}
```

Comment:
can we put it under the `Context` of line 97 ?
since its the same always, 
or there was some reason (i think) to not create stuff directly under Context ?

Reply:
I meant under the Context (which mean in the context) so it will still be on each context on its own.
but not crucial

Reply:
I think this must be in BeforeEach since it must be called after the BeforeEach in outer context which sets the `tcpVM`